app/js/controllers/actions/position/position-list-ctrl.js

```diff
@@ -20,9 +19,34 @@ angular
 		}	
 		$scope.data = data;
 		_.each($scope.data, function(element) {
+			element.attributes.responsibilities = element.attributes.responsibilities.length ==0 ? "None" : element.attributes.responsibilities.join(' ')
 			element.attributes.name = preProcess.positionToString(teamsIdToName, element, true);
 			element.attributes.team = preProcess.positionToString(teamsIdToName, element, false);
 			element.attributes.applicationForm = (element.relationships.applicationForm.data==null ? "None" : element.relationships.applicationForm);
 		});
 	});
+
+	$scope.updateSelection = function(resourceId) {
+		var index =	_.findIndex($scope.data, {'id': resourceId});
```

Comment:
minor: extra spaces before `_.findIndex`

Reply:
Made changes in commit 503cf38